modules/sdk/gradle-plugins-alloy-taglib/build.gradle

```diff
@@ -17,5 +17,5 @@ gradleTest {
 	dependsOn copyGradleTestDependencies
 	dependsOn jar
 
-	versions "2.5", "2.6", "2.7", "2.8", "2.9", "2.10", "2.11", "2.12", "2.13", "2.14.1", "3.0", "3.1", "3.2.1", "3.3", "3.5", "3.5.1"
+	versions "2.5", "2.6", "2.7", "2.8", "2.9", "2.10", "2.11", "2.12", "2.13", "2.14.1", "3.0", "3.1", "3.2.1", "3.3", "3.5", "3.5.1", "4.0", "4.0.1", "4.0.2", "4.1", "4.2", "4.2", "4.2.1", "4.3", "4.3.1", "4.4", "4.4.1", "4.5", "4.5.1", "4.6", "4.7", "4.8", "4.8.1", "4.9", "4.10", "4.10.1", "4.10.2"
```

Comment:
When we can reduce how far back we have to maintain compatibility :) especially since we usually ship gradle wrappers with all of our project templates.

Reply:
`Gradle 5.0` is a big change  with many APIs being removed so I was thinking of breaking backward compatibility at that time and only supporting 5.0.